crypto/s2n_hkdf.c

```diff
@@ -31,8 +31,9 @@
 int s2n_hkdf_extract(s2n_hmac_algorithm alg, const struct s2n_blob *salt, const struct s2n_blob *key, struct s2n_blob *pseudo_rand_key)
 {
     struct s2n_hmac_state hmac;
-    pseudo_rand_key->size = s2n_hmac_digest_size(alg);
-
+    uint8_t hmac_size;
+    GUARD(s2n_hmac_digest_size(alg, &hmac_size));
```

Comment:
This results in a size mismatch, `pseudo_rand_key->size` is `uint32_t`, and `hmac_size` is `uint8_t`